scripts/lint_yaml.py

```diff
@@ -412,6 +470,8 @@ def validate_person(self, person, filename, person_type, date=None):
         uid = person["id"].split("/")[1]
         if uid not in filename:
             self.errors[filename].append(f"id piece {uid} not in filename")
+        if not names_consistent(person):
```

Comment:
some background here: we used to have a warning for this, but turned it off as it wasn't useful and just created extra work with no real benefit.  before re-adding there'd need to be a good reason, the file name is generated to be human readable, and if the person's name changes a bit we can still understand who it applies to easily.  the id within the file is the source of truth, nothing in the filename should be taken as granted, enforcing this seems like a way around doing things via their name/id within the file

Reply:
There's a three-way data anomaly between the person name in `filename`, (yaml) `name`, and (yaml) `family_name` + `given_name`.

This validation check actually doesn't try to check the filename, but rather consistency between `name` and `given_name`/`family_name`.

I added it because it's very easy for a volunteers editting person records in airtable (probably in yaml files too) to change the name in one place but not the other. In fact, it happened numerous times in this first experiment with having volunteers edit records in airtable, which is what prompted me to add the check